packages/core/src/Account.ts

```diff
@@ -620,7 +627,7 @@ export class Account extends TypedEventEmitter<Events> {
     });
 
     const handleMissedNotifications = async (notificationId: string) => {
-      if (this.service?.mls) {
```

Comment:
now, the fact that the mls service is instantiated doesn't necesarily means the mls device is ready to use. We need to add some more checks before doing any kind of mls operation. 

This is due to the fact that we do not yet have the MLS config when the core is instantiated. So we cannot decide whether the MLS service should be instantiated at build time. Thus we need some extra checks